src/frontend/org/voltdb/RealVoltDB.java

```
@@ -2587,6 +2606,34 @@ private void stageDeploymentFileForInitialize(Configuration config, DeploymentTy
         ClusterSettings.create(CatalogUtil.asClusterSettingsMap(dt)).store();
     }
 
+    private void stageLicenseFile(Configuration config) {
+        // No such need in community version
+        if (!MiscUtils.isPro()) {
+            return;
+        }
+        String path = null;
+        try {
+            path = config.m_voltdbRoot.getCanonicalPath();
+        } catch (IOException e) {
+            VoltDB.crashLocalVoltDB("Unable to get voltdbroot path", false, e);
+        }
+        // delete the prior license if exists
+        File destF = new VoltFile(path, Constants.LICENSE_FILE_NAME);
+        if (config.m_forceVoltdbCreate && destF.exists()) {
+            destF.delete();
+        }
```

(1) if the file exists and it's not 'force', should we fail explicitly here rather than just letting the copy fail below?

(2)  rather than explicit delete, you could add a 'replace if exists' option to the copy call.

---

1) without --force, you can't re-initialize the already initialized voltdb root.
2) good idea.